### ISSUE TITLE
"Error: Status code: 429" Please help me

### ISSUE THREAD

**Issue by uguraltinsoy** [fent/node-ytdl-core#889]
OS : Centos 6 Linux
Node V = v14.16.0

```
npm ls miniget 

-- ytdl-core@4.5.0
  +-- m3u8stream@0.8.3
  | `-- miniget@4.2.0  deduped
  `-- miniget@4.2.0

```

(node:25232) UnhandledPromiseRejectionWarning: Error: Status code: 429
    at ClientRequest.httpLib.request (/root/Bots/MrMusic/node_modules/miniget/dist/index.js:190:31)
    at Object.onceWrapper (events.js:286:20)
    at ClientRequest.emit (events.js:198:13)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:565:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:111:17)
    at TLSSocket.socketOnData (_http_client.js:451:20)
    at TLSSocket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    at readableAddChunk (_stream_readable.js:269:11)
    at TLSSocket.Readable.push (_stream_readable.js:224:10)
(node:25232) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:25232) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by redbrain** [comment]
1) Make sure ytdl-core is the latest version
2) Use proxies (ex: https://github.com/fent/node-ytdl-core/blob/master/example/proxy.js)
3) Use cookies (ex: https://github.com/fent/node-ytdl-core/blob/master/example/cookies.js)
4) Wait it out (usually it goes away within a few days)
5) Switch to IPv6 since that gives you a free new address

Fixes courtesy Promise#0001 from the support Discord.

**Comment by uguraltinsoy** [comment]
I am using the latest version but unfortunately it has this problem since yesterday, windows 10 no such problem, only centos broke down in 1 day

**Comment by InspireVortex** [comment]
okay uhm, so im using this ytdl-core in my bot i code in Js, and it works properly for every single commands

**Comment by BR88C** [comment]
There's already an open issue, #635 , that goes into the issue in depth while also providing solutions.

**Comment by sdan41517** [comment]
v4.4.3 works fine but as soon as you upgrade to a newer version, it begins to throw error 429. Weird...

**Comment by redbrain** [comment]
Should this be closed as a dupe of #635?

**Comment by fent** [comment]
> v4.4.3 works fine but as soon as you upgrade to a newer version, it begins to throw error 429. Weird...

that is strange, the only difference in v4.4.4 was a json fix. no changes to requests or anywhere where it interacted with youtube

https://github.com/fent/node-ytdl-core/releases/tag/v4.4.4
